wp-content/themes/classicpress-susty-child/front-page.php

```diff
@@ -126,6 +113,20 @@
 	</div>
 </section>
 
+<!-- Changed to second position so as first we have the features/download section-->
+<section class="homepanel1">
```

Comment:
Not related to your PR, but this is why I hate css classes named where they are. 

It makes sense to reorder it, but now the class makes no sense since this is now `homepanel2` not `homepanel1` and both have specific css.

Reply:
I know - I thought of changing that but refrained since it’ll likely be a whole cascade 

should I check css too and change the classes?

Reply:
I think we may end up going down a rabbit hole, for now we can merge this and get it deployed. We will just add an issue to review it at some point 👍